### PR TITLE
executor,sessionctx: refine output format for tidb-specific variables of boolean type

### PR DESCRIPTION
**Pull request by spongedu**
### What problem does this PR solve? 

Currently, the result of selecting tidb-specific variables of boolean type is not well formatted. I think we'd better make it same style with MySQL. For example:
```sql
tidb> set @@tidb_batch_delete='On';
Query OK, 0 rows affected (0.00 sec)

tidb> select @@tidb_batch_delete;
+---------------------+
| @@tidb_batch_delete |
+---------------------+
| On                  |
+---------------------+
1 row in set (0.00 sec)

tidb> set @@tidb_enable_streaming='OfF'
    -> ;
Query OK, 0 rows affected (0.00 sec)

tidb> select @@tidb_enable_streaming;
+-------------------------+
| @@tidb_enable_streaming |
+-------------------------+
| OfF                     |
+-------------------------+
1 row in set (0.00 sec)

tidb> set @@tidb_disable_txn_auto_retry=on;
Query OK, 0 rows affected (0.00 sec)

tidb> select @@tidb_disable_txn_auto_retry;
+-------------------------------+
| @@tidb_disable_txn_auto_retry |
+-------------------------------+
| ON                            |
+-------------------------------+
1 row in set (0.00 sec)

tidb> set @@tidb_batch_insert=0;
Query OK, 0 rows affected (0.00 sec)

tidb> select @@tidb_batch_insert;
+---------------------+
| @@tidb_batch_insert |
+---------------------+
| 0                   |
+---------------------+
1 row in set (0.00 sec)

``` 

### Check List 

Tests 

 - Unit test
 - Integration test
 - Manual test (add detailed scripts or steps below)

Code changes

N / A

Side effects

N / A

Related changes

N / A

### CODE REVIEW

File: sessionctx/variable/varsutil.go [hunk 1]
```diff
@@ -374,9 +374,16 @@ func ValidateSetSystemVar(vars *SessionVars, name string, value string) (string,
...
 		return value, ErrWrongValueForVar.GenWithStackByArgs(name, value)
-	case GeneralLog, TiDBGeneralLog, AvoidTemporalUpgrade, BigTables, CheckProxyUsers, LogBin,
+	case TiDBSkipUTF8Check, TiDBOptAggPushDown,
+		TiDBOptInSubqToJoinAndAgg, TiDBEnableFastAnalyze,
+		TiDBBatchInsert, TiDBDisableTxnAutoRetry, TiDBEnableStreaming,
+		TiDBBatchDelete, TiDBBatchCommit, TiDBEnableCascadesPlanner, TiDBEnableWindowFunction,
+		TiDBCheckMb4ValueInUTF8, TiDBLowResolutionTSO, TiDBEnableIndexMerge, TiDBEnableNoopFuncs,
+		TiDBScatterRegion, TiDBGeneralLog, TiDBConstraintCheckInPlace:
+		fallthrough
+	case GeneralLog, AvoidTemporalUpgrade, BigTables, CheckProxyUsers, LogBin,
```
Comment (wshwsh12) [inline sessionctx/variable/varsutil.go]: I think you can make them in one `case`, to not use `fallthrough`.

Comment (spongedu) [reply sessionctx/variable/varsutil.go]: @wshwsh12 I think keep TiDB specific variables in a separate `case` is more clearer.  If put together, the merged `case` is too long to read, I think.

**Comment by jackysp** [comment]
Thanks @spongedu !

**Pull request merged**